src/properties/PropertiesImpl.ts

```diff
@@ -1,6 +1,13 @@
 import { MutableProperties, Properties, PropFlagVals } from "properties/Property";
 import SimpleMap from "interface/SimpleMap";
 import { requireNotNull, isDefined } from "util/Utils";
+import { asString } from "util/AsFunctions";
+
+const AKey = {
```

Comment:
What does the "A" in AKey refer to?  Given that minification will rename this anyways, would a full word be more readable?

Reply:
Has been changed to const AttribKey